docs/guide/release-notes.asciidoc

```diff
@@ -0,0 +1,261 @@
+  `using Asyncio with {es} 
+  (https://github.com/elastic/elasticsearch-py/pull/1232[#1232], 
+  https://github.com/elastic/elasticsearch-py/pull/1235[#1235], 
+  https://github.com/elastic/elasticsearch-py/pull/1236[#1236]).
+* Added async helpers `async_bulk`, `async_streaming_bulk`, `async_scan`, and 
+  `async_reindex` 
+  (https://github.com/elastic/elasticsearch-py/pull/1260[#1260]).
+* Updated `exists_source` API to use non-deprecated {es} API routes when 
+  `doc_type` is not specified to suppress deprecation warnings 
+  (https://github.com/elastic/elasticsearch-py/pull/1272[#1272]).
+
+
+[discrete]
+[[rn-771]]
+=== 7.7.1 (2020-05-26)
+
+* Updated `create`, `update`, `explain`, `get_source`, and `termvectors` APIs to 
+  use non-deprecated {es} API routes when `doc_type` is not specified to 
+  suppress deprecation warnings 
+  (https://github.com/elastic/elasticsearch-py/pull/1253[#1253]).
+
+
+[discrete]
+[[rn-770]]
+=== 7.7.0 (2020-05-13)
+
+* Added support for ES 7.7 APIs 
+  (https://github.com/elastic/elasticsearch-py/pull/1182[#1182]).
+* Added `ElasticsearchDeprecationWarning` which is raised when a `Warning` HTTP 
+  header is sent by {es} 
+  (https://github.com/elastic/elasticsearch-py/pull/1179[#1179]).
+* Added support for serializing `numpy` and `pandas` data types to 
+  `JSONSerializer` 
+  (https://github.com/elastic/elasticsearch-py/pull/1180[#1180]).
+* Added `certifi` as a dependency so HTTPS connections work automatically.
+* Fixed duplicated parameters in some API docstrings 
+  (https://github.com/elastic/elasticsearch-py/pull/1169[#1169], thanks to 
+  https://github.com/mortenhauberg[Morten Hauberg]).
+
+
+[discrete]
+[[rn-760]]
+=== 7.6.0 (2020-03-19)
+
+
+* Added support for ES 7.6 APIs.
+* Added support for 
+  https://www.elastic.co/guide/en/elasticsearch/reference/current/tasks.html#_identifying_running_tasks[`X-Opaque-Id`] 
+  to identify long-running tasks.
+* Added support for HTTP compression to `RequestsHttpConnection`.
+* Updated default setting of `http_compress` when using `cloud_id` to `True`.
+* Updated default setting of `sniffing` when using `cloud_id` to `False`.
+* Updated default port to `443` if `cloud_id` and no other port is defined on 
+  the client or within `cloud_id`.
+* Updated `GET` HTTP requests that contain a body to `POST` where the API allows 
+  this to fix proxies rejecting these requests.
+* Fix regression of `client.cluster.state()` where the default `metric` should 
+  be set to `"_all"` if an index is given 
+  (https://github.com/elastic/elasticsearch-py/pull/1143[#1143]).
+* Fix regression of `client.tasks.get()` without a `task_id` having similar 
+  functionality to `client.tasks.list()` This will be removed in `v8.0` of 
+  `elasticsearch-py` 
+  (https://github.com/elastic/elasticsearch-py/pull/1157[#1157]).
+
+
+[discrete]
+[[rn-751]]
+=== 7.5.1 (2020-01-19)
+
+* `7.5.0` tag was not released so retagging.
+
+
+[discrete]
+[[rn-750]]
```

Comment:
Let's omit the 7.5.0 release and roll everything into 7.5.1 because effectively there was no 7.5.0 "release"

Reply:
Omitted via b7e33a8